storage/memory/memory_test.go

```diff
@@ -281,10 +301,48 @@ func TestObjects(t *testing.T) {
 		}
 	}
 	if cnt != 3 {
-		t.Errorf("g.Objects(%s, %s) failed to retrieve 3 objects, got %d instead", ts[0].Subject(), ts[0].Predicate(), cnt)
+		t.Errorf("g.Objects(%s, %s) failed to retrieve 3 objects, got %d instead ", ts[0].Subject(), ts[0].Predicate(), cnt)
 	}
 }
 
+func TestObjectsOffset(t *testing.T) {
```

Comment:
Would you mind adding a comment explaining the goal of the test and what you expect?

Reply:
Done